components/salesforce/salesforce.app.js

```diff
@@ -11,12 +11,25 @@ module.exports = {
     _instance() {
       return this.$auth.yourinstance;
     },
+    _instanceUrl() {
+      return this.$auth.instance_url;
+    },
+    _subdomain() {
+      return (
+        this._instance() ||
+        this._instanceUrl()
+          .replace('https://', '')
+          .replace('.salesforce.com', '')
```

Comment:
just to confirm, based on what you know, we should include the full subdomain in all cases, e.g. `toptal3-dev-ed.my` when you have a "my" portion of the domain, and just `na174` for `na174.salesforce.com`?

Reply:
Yup, that's correct. The "instance" ID is just a generic subdomain that users can use to call the API, and in this context it's no different than any other `custom-subdomain.with-multiple-hierarchies.salesforce.com`